src/app/content/search/hooks.spec.ts

```diff
@@ -310,7 +312,7 @@ describe('hooks', () => {
       );
     });
 
-    it('dispatches REPLACE with search query when page is undefined', () => {
+    it('dispatches PUSH with search query when page is undefined', () => {
```

Comment:
@TomWoodward this how it should be, right?

Reply:
whether the page is defined or not isn't the criteria, the way we have it set up it should be a `push` if the url location is changing, or a `replace` if it is only modifying the query.

so if you're on a 404 page and you do a search that has no results, that would be a replace still, but if it has results and it goes somewhere it'd `push`